src/julia.h

```diff
@@ -460,6 +460,7 @@ typedef struct {
     // `wrapper` is either the only instantiation of the type (if no parameters)
     // or a UnionAll accepting parameters to make an instantiation.
     jl_value_t *wrapper;
+    jl_value_t *Typeofwrapper;
```

Comment:
Since you are mutating this field, it needs to be marked atomic (release/consume):
```suggestion
    _Atomic(jl_value_t*) Typeofwrapper;
```